README.md

```diff
@@ -1,68 +1,49 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
nit: If this README is still valid for frontend, you can move it to frontend/

Reply:
Just to clarify, does this mean I should have the README in the main folder as well another copy of it in the frontend/ folder?